src/com/dotcms/api/system/event/verifier/PermissionVerifier.java

```diff
@@ -0,0 +1,32 @@
+package com.dotcms.api.system.event.verifier;
+
+
+import com.dotcms.api.system.event.Payload;
+import com.dotcms.api.system.event.PayloadVerifier;
+import com.dotcms.exception.BaseRuntimeInternationalizationException;
+import com.dotcms.rest.api.v1.system.websocket.SessionWrapper;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.PermissionAPI;
+import com.dotmarketing.business.Permissionable;
+import com.dotmarketing.exception.DotDataException;
+
+
+public class PermissionVerifier implements PayloadVerifier{
+
+    private final PermissionAPI permissionAPI;
+
+    public PermissionVerifier(){
```

Comment:
Use the constructor approach we used to use for the dependency.
One with Testing visibility that receives the instance per parameter and the default just sending the instance.
such as

this (APILocator.getPermissionAPI());